caddy/Caddyfile

```diff
@@ -7,7 +7,11 @@
 
     reverse_proxy /auth* http://keycloak:8080
 
-    reverse_proxy /mailhog* http://mailhog:8025
+    redir /mailpit /mailpit/
+    handle_path /mailpit/* {
+        rewrite * /mailpit{path}
+        reverse_proxy mailpit:8025
+    }
```

Comment:
```suggestion
    reverse_proxy /mailpit* http://mailpit:8025
```

Could you please test this configuration first. It's simpler and easier to maintain. :shrug: 
The [caddy docu](https://caddyserver.com/docs/caddyfile/directives/reverse_proxy) doesn't state any different.

Reply:
@derrabauke  it's working with your suggestion too